packages/dev/core/src/Cameras/arcRotateCamera.ts

```diff
@@ -1033,10 +1033,12 @@ export class ArcRotateCamera extends TargetCamera {
             this.radius += (this._goalRadius - this.radius) * t;
             Vector2.LerpToRef(this.targetScreenOffset, this._goalTargetScreenOffset, t, this.targetScreenOffset);
 
+            TmpVectors.Quaternion[0].negateToRef(TmpVectors.Quaternion[1]);

```

Comment:
Just thinking about the usage here - did you mean `negate` or `inverse`?

Reply:
Negate because of double cover : https://www.reedbeta.com/blog/why-quaternions-double-cover/